services/karma/2.0.1/defaults/cm.yaml

```diff
@@ -76,7 +76,7 @@ data:
           prefix: /dkp/kommander/monitoring/karma/
         log:
           config: true
-          level: warn
+          level: info
```

Comment:
Is `warn` not a real level?

Reply:
i wonder if it's `warning`

Reply:
it does look like `warning` is the right setting 

https://github.com/prymitive/karma/blob/main/docs/CONFIGURATION.md#log

Reply:
we will just revert to info, these were not generating so much noise to cause disk issues

Reply:
fixed in #550 for `main` branch.